src-docs/src/views/tool_tip/tool_tip_example.js

```diff
@@ -17,10 +17,20 @@ import {
 import ToolTip from './tool_tip';
 const toolTipSource = require('!!raw-loader!./tool_tip');
 const toolTipHtml = renderToHtml(ToolTip);
+const tooltipSnippet = `<EuiToolTip position="top" content="Here is some tooltip text">
+  <!-- An inline element to trigger the tooltip -->
+</EuiToolTip>
+`;
 
 import IconTip from './icon_tip';
 const infoTipSource = require('!!raw-loader!./icon_tip');
 const infoTipHtml = renderToHtml(IconTip);
+const infoTipSnippet = `<EuiIconTip
+  content="Here is another tooltip text"
```

Comment:
Just because both snippets aren't necessarily seen together...
```suggestion
  content="Tooltip text for the icon"
```